provisions/roles/jenkins/slave/tasks/main.yml

```diff
@@ -37,3 +37,10 @@
     - jenkins
     - monitor
     - jenkins/slave
+
+- name: Set PYTHONPATH environment variable
+  lineinfile:
+      dest: /etc/environment
+      state: present
+      regexp: '^PYTHONPATH'
+      line: 'PYTHONPATH=/opt/cccp-service'
```

Comment:
Please make sure this is appending path `/opt/cccp-service` to `PYTHONPATH`'s value. Other modules, if in future tries to append value to it, it could be a conflict.